sklearn/impute/_base.py

```diff
@@ -456,6 +463,12 @@ def _sparse_fit(self, X, strategy, missing_values, fill_value):
                     elif strategy == "most_frequent":
                         statistics[i] = _most_frequent(column, 0, n_zeros)
 
+                    elif isinstance(strategy, Callable):
+                        statistics[i] = self.strategy(column)
+
+                    else:
+                        raise RuntimeError(f"Unknown strategy {strategy}")
```

Comment:
I don't think this can ever happen since we validate parameters before getting here and people shouldn't be calling these private methods themselves.

Reply:
I was unable to get code coverage to pass without adding the final condition and this test.

Reply:
If you remove this line, codecov doesn't have them to want to cover in the first place.